### DISCUSSION TITLE
too vivid,not natural, like looking at lots of filters on mac

### DISCUSSION THREAD

**Discussion by tonho911** [smplayer-dev/smplayer#867] [Q&A] [answered]
I downloaded the file from this link to desktop https://www.youtube.com/watch?v=kF-0q042Jjk
to compare the color saturation of the results between SMPlayer on Windows and Mac and the results on Windows are very good but why is it too vivid,not natural, like looking at lots of filters on Mac or whatever it's called.
How do I set it so that the resulting colors are like Windows 10?

**Comment by smplayer-dev** [comment]
I see a small difference compared to the video played on the youtube site.
Have you tried adjusting the color in the video equalizer?

**Reply by tonho911** [reply]
Yes, I've tried adjusting the colors on the video equalizer, but what I get is that the settings I made are not the same in all the videos in the playlist, for example in one video the video equalizer is according to what I want, the second video and the three video settings return to default before I adjust them.
where can I set the video equalizer just once for all the videos I watch and it is saved?
Thank you for your attention sir

**Reply by tonho911** [reply]
I have tried the best thing in my opinion (adjusting the color) by SMPlayer->preferences->video->output driver->libmpv, but here other problems arise, abnormal video size, mouse seeking settings, next and so on doesn't work 'n fail...

or I can adjust to user defined but how do I do that, sir?

**Reply by smplayer-dev** [reply, marked as answer]
> where can I set the video equalizer just once for all the videos I watch and it is saved?

In the video equalizer click on the "set as default values" button.
